Firestore/core/src/firebase/firestore/remote/remote_objc_bridge.cc

```diff
@@ -52,7 +52,6 @@ using nanopb::Message;
 using nanopb::Reader;
 using remote::ByteBufferReader;
 using remote::Serializer;
-using util::MakeString;
```

Comment:
Hmm, would it be possible to configure Clang-tidy to catch unused `using` declarations? (not for this PR)

Reply:
I have yet to find a way to make `clang-tidy` available for CI, short of building and distributing our own binary. My attempts to build my own binary have been stymied by https://github.com/include-what-you-use/include-what-you-use/issues/100: clang-tidy can't find any headers available in the SDK leading to thousands of errors building and no useful feedback.